config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
 
   # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
-  # config.require_master_key = true
+  config.require_master_key = true
```

Comment:
Question: Is this line necessary? Do you need it to make credentials work?

Reply:
removed